src/services/NurseryInventoryService.ts

```diff
@@ -67,6 +71,7 @@ export type SearchInventoryParams = {
   searchSortOrder?: SearchSortOrder;
   query?: string;
   facilityIds?: number[];
+  isDowloading?: boolean;
```

Comment:
typo isDowloading, or maybe call it isCsvExport?

Reply:
Fixed!